daemon/daemon.go

```diff
@@ -109,6 +111,8 @@ type Daemon struct {
 	defaultLogConfig runconfig.LogConfig
 	RegistryService  *registry.Service
 	EventsService    *events.Events
+	uidMaps          []idtools.IDMap
```

Comment:
Read from this comment on down.. https://github.com/docker/libcontainer/pull/554#issuecomment-96083942 

I'm happy to do either, but it would definitely be nice to not have many copies of the exact same struct.